conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -56,7 +56,7 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artifacts.
 conda clean --lock
 
-conda install --yes --quiet conda-forge-ci-setup=1
+conda install --yes --quiet conda-forge-ci-setup=1 conda-build
```

Comment:
Maybe we should break this out into a `conda update` call? That said, this would defeat `conda-forge-ci-setup` a bit. What if we bump the version constraint in `conda-forge-ci-setup` to ensure an update?

Reply:
I *think* this has the same effect as bumping the constraint in ci-setup without having to actually publish a new ci-setup for each conda-forge release. Will do a little testing.